uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/style/_inputs.scss

```diff
@@ -0,0 +1,5 @@
+// removes fixed with for inputs added by CiviCRM:
+input, textarea, select {
+  width: auto !important;
```

Comment:
How can we make sure this doesn't break anything in other places?

Reply:
These inputs for CiviCRM are always 230px in length. This breaks two things:
1. inputs can't adapt to their parent width.
2. can't have smaller inputs, all of them are 230px wide.

Why this code shouldn't affect other places:

This is wrapped inside of the bootstrap theme id like this:
```scss
#bootstrap-theme input, #bootstrap-theme textarea, #bootstrap-theme select {
  width: auto !important;
}
```
So that means it won't affect anything outside bootstrap.

`width: auto` is the default behaviour for bootstrap inputs: 
https://github.com/twbs/bootstrap-sass/blob/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/stylesheets/bootstrap/_forms.scss#L503

If an input needs to be of a specific length the size attr can be used. 

If an input needs to be of a specific pixel width then yes, this might affect those types of inputs. I did a quick look around the app and nothing seems to be breaking.

If this is a concern we could wrap this for the admin by adding an extra class.

Reply:
Awesome. Thank you for the explanation.

Reply:
To document the decision: it was decided that this is the only way to go since CRM sets 230px !important.